consul/consul.go

```diff
@@ -145,11 +145,11 @@ func buildServiceKey(service *bridge.Service, key string) string {
 }
 
 func (r *ConsulAdapter) Deregister(service *bridge.Service) error {
-	r.removeAttributes(service)
+	r.RemoveAttributes(service)
```

Comment:
`r.RemoveAttributes(service)` should be removed from here. It is called conditionally in`remove(containerId string, deregister bool)` of bridge.go